deny.toml

```diff
@@ -0,0 +1,37 @@
+# https://embarkstudios.github.io/cargo-deny/
+
+targets = [
+    { triple = "aarch64-apple-darwin" },
+    { triple = "aarch64-linux-android" },
+    { triple = "x86_64-apple-darwin" },
+    { triple = "x86_64-pc-windows-msvc" },
+    { triple = "x86_64-unknown-linux-gnu" },
+    { triple = "x86_64-unknown-linux-musl" },
+]
+
+
+[advisories]
+vulnerability = "deny"
+unmaintained = "warn"
+yanked = "deny"
+ignore = [
+    "RUSTSEC-2020-0071", # https://rustsec.org/advisories/RUSTSEC-2020-0071 - Potential segfault in the time crate.
+    "RUSTSEC-2020-0159", # https://rustsec.org/advisories/RUSTSEC-2020-0159 - Potential segfault in `localtime_r` invocations.
+]
+
+
+[bans]
+multiple-versions = "deny"
+wildcards = "allow" # at least until https://github.com/EmbarkStudios/cargo-deny/issues/241 is fixed
+deny = []
+skip = []
+skip-tree = [
+    { name = "criterion" },  # dev-dependency
+    { name = "quickcheck" }, # dev-dependency
+]
+
+
+[licenses]
+unlicensed = "allow"
+allow-osi-fsf-free = "either"
+copyleft = "allow"
```

Comment:
This is for `MPL-2.0` only, right? I'm not sure if we rather give a specific exception as strong copyleft (GPL) is definitely _not_ intended to be required to build `image` at this moment. The jury is still out on whether building Rust crates compliant with LGPL's linking exception is actually legally tested and possible.

Reply:
This allows any and all licenses. If we want to deny GPL we can revert https://github.com/image-rs/image/pull/1698/commits/fec1d0286b3ef73e93253127d9b85e1ec7a979d3 (perhaps in a separate PR?)